### ISSUE TITLE
First layer above zig-zag support sometimes is parallel to support lines

### ISSUE THREAD

**Issue by vlachoudis** [Ultimaker/Cura#5129]
Just a minor issue, I use the zig-zag support structures and sometimes the layer just above the structures is done with lines which are parallel to the support lines resulting in long bridges in between the structures.

**Application Version**
3.6

**Platform**
Windows 10

**Comment by vlachoudis** [comment]
Of course, what information you would need?
I've uploaded the STL file in dropbox https://www.dropbox.com/s/e8szbebx4ocxmuj/Sample.stl?dl=0 
The cura profile https://www.dropbox.com/s/4pupnlb14cth18n/mygood.curaprofile?dl=0 
and two screenshots on layers 37 and 38 
![image](https://user-images.githubusercontent.com/3834594/50850049-17a26500-1379-11e9-8e70-a98feb3e92a5.png)
![image](https://user-images.githubusercontent.com/3834594/50850065-1ffaa000-1379-11e9-954a-447ff6914b2c.png)

When I load the STL I rotate it on Y (green) by -180deg
The settings are
Quality: Heigh: 0.2, Initial layer 0.3,
Shell: Wall, Thickness 0.8, Top/Bottom layers 5
Support: Zig Zag, Placement Everywhere, Density 15%
Build: Skirt, Count 2, Distance 3, Minimum length 250mm

**Comment by ctmakro** [comment]
this may be the same problem as #4538

**Comment by smartavionics** [comment]
> this may be the same problem as #4538

I don't think so, that was related to bridging but here we are looking at support.

I'm not an expert at support but it seems to me that if you have low support density and are not using the support interface (aka roof) then you can expect the skin and/or wall lines to fall through the gaps in the support. Isn't that what the interface is for?

**Comment by gloomytree** [comment]
I have that problem, too. Any updates? Cura generated a 10 cm bridge with Zig Zag. Density had no effect, even 40% generated the bridge.

**Comment by Ghostkeeper** [comment]
If you enable the extra bridge settings, there is a setting called "Bridge Skin Support Threshold". Setting that to 0 should prevent it from generating bridges, as a workaround.
However there is a different bug with that setting, too, where setting it to 0 is not working properly, so you should set it to 0.01 instead or something.

This bug where it's still bridging while the bridge is supported with support still stands, though. We didn't get around to working on this yet.

**Comment by gloomytree** [comment]
@Ghostkeeper Ty, but a user might small bridges in other parts of the model. Also, I'm pretty sure the bridging settings don't affect support material.
